locale/en/blog/vulnerability/september-2019-openssl-updates.md

```diff
@@ -0,0 +1,53 @@
+---
+date: 2019-09-05T15:34:35.000Z
+category: vulnerability
+title: OpenSSL upgrade low-severity Node.js security fixes
+slug: openssl-and-low-severity-fixes-sep-2019
+layout: blog-post.hbs
+author: Sam Roberts
+---
+
+### Summary
+
+The Node.js project may be releasing new versions across all of its active
```

Comment:
I wonder if it would be clearer to refer to them as "supported release lines" rather than "active release lines" but that's a bigger discussion than this one post and should not hold it up.

Reply:
> 
> 
> I wonder if it would be clearer to refer to them as "supported release lines" rather than "active release lines" but that's a bigger discussion than this one post and should not hold it up.

See https://github.com/nodejs/Release/issues/359 for where that discussion is taking place. Rod says (https://github.com/nodejs/Release/issues/359#issuecomment-448077853): 
> I say "actively supported release lines" whenever I do security releases that need to cover them all.

Reply:
"actively-supported" would be 👍 by me. Although it does kind of raise the question of what release lines are "passively-supported" so...  Words are difficult.

Reply:
supported makes sense to me, changed it.